code/frontend/src/styles/_themes.scss

```
@@ -95,10 +95,64 @@
 
   // Ambient orbs
   --orb-opacity: 0.15;
-  --orb-blur: 120px;
-  --orb-primary: radial-gradient(circle, var(--brand-500), var(--brand-800));
-  --orb-secondary: radial-gradient(circle, var(--brand-400), var(--brand-600));
-  --orb-tertiary: radial-gradient(circle, var(--brand-400), var(--brand-700));
+  --orb-1: #6b34cb;
+  --orb-1: color-mix(in srgb, var(--brand-800) 67%, var(--brand-500));
```

Stylelint (17.14.0)

[error] Duplicate custom property "--orb-1" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)